pom.xml

```diff
@@ -1011,7 +1011,7 @@
             <exclude>kubernetes-client/src/test/resources/mockito-extensions/*</exclude>
             <exclude>**/src/test/resources/ssl/*</exclude>
             <exclude>**/src/test/resources/io/fabric8/java/generator/approvals/*.*.approved.txt</exclude>
-            <exclude>**/java-generator/it/**/expected/**/*</exclude>
+            <exclude>**/it/**/expected/**/*.*</exclude>
```

Comment:
I would keep the /java-generator/ part just in case.

Reply:
Not keeping `/java-generator/` makes this command unusable with e.g.:
```
mvn license:format -f java-generator/pom.xml
```
(which is my workflow)
What do you think about renaming `AuthSpec.java` -> `AuthSpec.expected` and changing this line to:
```xml
<exclude>**/it/**/expected/**/*.expected</exclude>
```

Reply:
I'm not following (too many double negations) your point.

> What do you think about renaming AuthSpec.java -> AuthSpec.expected and changing this line to:

This is fine too. The important thing here is that we don't "overexclude" stuff.

Reply:
done